src/BizHawk.Emulation.Cores/Consoles/Nintendo/NES/Boards/Mapper227.cs

```diff
@@ -64,12 +64,12 @@ public override void WritePrg(int addr, byte value)
 			//maybe only the multicarts do it, to keep the game from clobbering vram on accident
 			//vram_protected = o;
 
-			if (o == true && S == false)	
+			if (o && S == false)	

```

Comment:
I'm not sure exactly how I'd do it because it looks like it could get a bit indent pyramid of doom-like but it might be more readable/efficient to group these conditions instead of rechecking the whole condition every if line